.ci/containers/gcb-terraform-vcr-tester/test_terraform_vcr.sh

```diff
@@ -57,8 +57,20 @@ update_status "pending"
 set +e
 # cassette retrieval
 mkdir fixtures
-gsutil -m -q cp gs://ci-vcr-cassettes/beta/fixtures/* fixtures/
-# copy branch specific cassettes over master. This might fail but that's ok if the folder doesnt exist
+# # the check of existence of BASE_BRANCH is for backwards-compatability
+if [ -n "$BASE_BRANCH" ]; then
+  if [ "$BASE_BRANCH" != "FEATURE-BRANCH-major-release-5.0.0" ]; then
+    # pull main cassettes (major release uses branch specific casssettes as primary ones)
```

Comment:
Since we'll make changes that may affect a large surface of providers and may end up have cassettes need to be re-recorded in major release PRs if we have branch cassettes on top of the main cassettes. More details can be seen in the design doc